generator/crds/gen.go

```diff
@@ -158,11 +165,30 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 			} else {
 				fileName = fmt.Sprintf("%s_%s.%s.yaml", crdRaw.Spec.Group, crdRaw.Spec.Names.Plural, crdVersions[i])
 			}
-			if err := ctx.WriteYAML(fileName, crd); err != nil {
+			if err := ctx.WriteYAML(fileName, extCrd); err != nil {
 				return err
 			}
 		}
 	}
 
 	return nil
 }
+
```

Comment:
Why is this necessary now ?

Reply:
You mean filterTypesForCRDs but not a new line, right?

Reply:
If `filterTypesForCRDs` probably we could filter nothing, but I thought it would be a good idea to do the same as controller-tools for CRDs does.

Reply:
I can revert it and we'll add it only when it will be needed.

Reply:
removed